### PR TITLE
Added Adopter's Pages

### PR DESCRIPTION
**Pull request by hrishavjha**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,11 +24,17 @@
     "update": "ncu -u"
   },
   "dependencies": {
+    "@mdx-js/mdx": "^1.6.22",
+    "@mdx-js/react": "^1.6.22",
+    "@reach/router": "^1.3.4",
```
Comment (arkajyotiMukherjee) [inline package.json:29]: reach router and it's types are not needed to be installed, they comes with gatsby. You can imort them from gatsby itself

File: src/components/image-fetch/AdopterImage.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { graphql, useStaticQuery } from "gatsby";
+import Img from "gatsby-image";
```
Comment (arkajyotiMukherjee) [inline src/components/image-fetch/AdopterImage.tsx:2]: Since you have already used the new `gatsby-plugin-image` and the old `gatsby-image` is no more required.
This will do auto migration: https://www.gatsbyjs.com/plugins/gatsby-plugin-image/#migrating
After migration, if you feel some images are static like this one use the appropriate `<StaticImage />` component: https://www.gatsbyjs.com/plugins/gatsby-plugin-image/#static-images

File: src/components/sections/adopters/Adopters.data.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+const AdopterList = [
+  {
+    reverse: true,
+    bgColor: "#000000",
```
Comment (arkajyotiMukherjee) [inline src/components/sections/adopters/Adopters.data.tsx:4]: refer to reviews in the old PR #156

File: src/components/sections/individual-adopters/Video.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 10;
+`;
+
+const PlayButton = styled.div`
+  // position: absolute;
+  // top: 0;
+  // left: 0;
```
Comment (arkajyotiMukherjee) [inline src/components/sections/individual-adopters/Video.tsx:53]: removed commented code

File: src/components/sections/adopters/Adopters.data.tsx (hunk 3 above)
Comment (arkajyotiMukherjee) [reply src/components/sections/adopters/Adopters.data.tsx:4]: same as individual-adopters-data.ts file

File: src/pages/adopters/anutanetworks.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+import { useLocation } from "@reach/router";
+
+const AnutaNetworks = () => {
+  const location = useLocation();
+  const urlSlug = location.pathname
+    ?.replace(/\/$/, "")
+    .split("/")
+    .pop();
+  return (
+    <Layout noPreFooter>
```
Comment (arkajyotiMukherjee) [inline src/pages/adopters/anutanetworks.tsx:15]: you can make these with the createPage API dynamically and use this as template

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]
Inside static folder:
- there is another static folder, remove that...that is auto-generated
- folder name `Banner` to `banner`

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]
file: sections/adopters/Adopters.data.tsx 
 rename to sections/adopters/adopters-data.ts
 and inside make the constant `adoptersList` in line no 1

File: src/pages/adopters/anutanetworks.tsx (hunk 5 above)
Comment (Aryan007007) [reply src/pages/adopters/anutanetworks.tsx:15]: Arko I already did made it using the dynamic createPage Api, but I think this force push to staging mixed up a lot of code.
Im trying to clean everything up

File: src/components/sections/adopters/ListAdopter.tsx [hunk 6]
```diff
@@ -74,31 +69,41 @@ const CompanyInfoCardText = styled.p`
...
+        </ResponsiveRow>
+      </Center>
     </>
   );
 };
-const ListAdopters: React.FC<IProps> = ({ url }) => {
-  return (
-    <>
+const ListAdopters: React.FC<IProps> = ({ url }) => (
+  <>
```
Comment (arkajyotiMukherjee) [inline src/components/sections/adopters/ListAdopter.tsx:105]: fragment is not needed here

File: src/pages/index.tsx [hunk 7]
```diff
@@ -24,6 +25,12 @@ const App = () => {
       <FeedBack />
     </Layout>
   );
+
+  return (
+    <Router>
+      <Home path="/" />
```
Comment (arkajyotiMukherjee) [inline src/pages/index.tsx:31]: why is this required?
gatsby automatically puts index into `/` path

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Aryan007007) [reply package.json:29]: So we're keeping reach router !

File: src/components/image-fetch/AdopterImage.tsx (hunk 2 above)
Comment (Aryan007007) [reply src/components/image-fetch/AdopterImage.tsx:2]: I've successfully switched over to gatsby-plugin-image

File: src/pages/index.tsx (hunk 7 above)
Comment (Aryan007007) [reply src/pages/index.tsx:31]: Removed, I was trying something and then didn't need it !

**Comment by Aryan007007** [comment]
The build failed for this reason : 
Gatsby requires Node.js 12.13.0 or higher (you have v10.19.0).

But my machine is on v12.19.0

**Review by arkajyotiMukherjee — APPROVED** [review]

**Comment by arkajyotiMukherjee** [comment]
@Aryan007007 use the Existing VideoFrame component for the Individual Adoptors page
